app/schemas/models/cinematic.schema.js

```diff
@@ -0,0 +1,151 @@
+const c = require('./../schemas')
+
+const CharacterSchema = (title) => c.object({
+  title: title,
+  description: 'ThangType that will appear on the right side of the screen.',
```

Comment:
I believe it is a generic character schema for thang types that will appear in the cinematics screen. Can change the description to be generic instead of specific to `right side`.